src/Microsoft.AspNet.Mvc.Core/Rendering/Html/DefaultEditorTemplates.cs

```diff
@@ -374,6 +374,22 @@ public static string NumberInputTemplate(IHtmlHelper htmlHelper)
             return GenerateTextBox(htmlHelper, inputType: "number");
         }
 
+        public static string FileInputTemplate(IHtmlHelper htmlHelper)
+        {
+            return GenerateTextBox(htmlHelper, inputType: "file");
+        }
+
+        public static string FileCollectionInputTemplate(IHtmlHelper htmlHelper)
+        {
+            var htmlAttributes = new Dictionary<string, object>()
+            {
+                { "multiple", "" }
```

Comment:
if this remains (not sure it's needed w/ change below), value must be "multiple" and not the empty string.  we generate XHTML-compliant HTML.